src/main/scala/castalia/JsonConverter.scala

```diff
@@ -0,0 +1,31 @@
+package castalia
+
+import java.io.FileNotFoundException
+import java.net.URL
+
+import spray.json._
+
+import scala.util.Try
+
+/**
+  * Created by jens on 13-12-15.
+  */
+object JsonConverter {
+  def parseJson[T: JsonReader](filename: String): T = {
+    val resource: URL = getClass.getResource("/" + filename)
+    resource match {
+      case url : URL => unmarshalJsonToClass[T](url) getOrElse
+        (throw new UnmarshalException(s"Type could not be unmarshelled from $url"))
+      case _ => throw new FileNotFoundException(filename)
```

Comment:
Getresources Returns:
A URL object or null if no resource with this name is found.
Using getresourceasstream() throws a nullpointerexception when the resource doesnt exist. So it's matter of taste.